2-mandatory/3-stocks.js

```diff
@@ -64,7 +81,24 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
```

Comment:
Good work on this one 👍 
In case you're interested, JavaScript also gives us something you might find useful here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max